### ISSUE TITLE
Version has been deprecated and shows up with an obsoleted error

### ISSUE THREAD

**Issue by rseymour** [docker/docs#19684]
### Is this a docs issue?

- [X] My issue is about the documentation content or website

### Type of issue

Information is incorrect

### Description

docker compose now reports that the `version` key is obsolete after its 10 year existence. The docs still suggest setting it.

### Location

https://docs.docker.com/compose/compose-file/

### Suggestion

Warn that things change, even after a decade of not changing, sometimes before the docs do.

**Comment by rseymour** [comment]
Optional is one word and implies the presence or absence won't be noted by any code. The _new_ log message uses the word *obsolete* and appeared out of nowhere. Creating a small flurry of issues in other repos both open and closed. Logging something as obsolete is not the same as the former state of not logging and treating it as optional. 

```
docker-compose.yml: `version` is obsolete
```

**Comment by astehlik** [comment]
I agree. Either the "obsolete" warning needs to be removed again from the code or the docs need adjustment.

Currently it is not clear from the docs that using `version` causes a warning.

**Comment by rseymour** [comment]
Not to mention the fact that the "`version` is obsolete" message could be misread as the version you've specified is now obsolete. I'm not sure how this crept in, but it's disheartening to see an issue with the output of the code mismatching the documentation closed as completed.
